src/ansible_navigator/configuration_subsystem/navigator_configuration.py

```diff
@@ -483,5 +501,13 @@ class Internals(SimpleNamespace):
             ),
             value=SettingsEntryValue(),
         ),
+        SettingsEntry(
+            name="workdir",
+            cli_parameters=CliParameters(short="--bwd"),
+            settings_file_path_override="builder.workdir",
+            short_description="Specify the path that contains ansible-builder manifest files",
+            subcommands=["builder"],
+            value=SettingsEntryValue(default=os.getcwd()),
```

Comment:
(important) Should this be the root of the project where the navigator config is present (or `.git/`, for example)?

Reply:
If the navigator config file is present in CWD it takes precedence over other shared paths. Also after developing collection if the user wants to build EE chances are the builder manifest files will most likely be present in the CWD itself.

Reply:
I like this default.